packages/account/src/providers/provider.ts

```diff
@@ -674,8 +674,9 @@ Supported fuel-core version: ${supportedVersion}.`
   private createOperations(): SdkOperations {
     const fetchFn = Provider.getFetchFn(this.options);
     const gqlClient = new GraphQLClient(this.urlWithoutAuth, {
-      fetch: (url: string, requestInit: RequestInit) => fetchFn(url, requestInit, this.options),
-      responseMiddleware: (response: GraphQLResponse<unknown> | Error) => {
+      fetch: (input: RequestInfo | URL, requestInit?: RequestInit) =>
+        fetchFn(input.toString(), requestInit || {}, this.options),
+      responseMiddleware: (response: GraphQLClientResponse<unknown> | Error) => {
```

Comment:
This is all internal and adds no breaking change, right?

Reply:
Correct 👍🏻